bridge/image.yaml

```diff
@@ -42,7 +42,6 @@ packages:
 
 run:
   workdir: "/opt/strimzi"
-  user: 185
```

Comment:
I think this user id allows users to create files in the bridge container. We don't need to ever create files in the bridge container, right?

Reply:
no we don't. I run and used the bridge in this way and it worked fine.